ui/src/components/billing/BillingDialogPaymentMethod.vue

```diff
@@ -23,10 +23,19 @@
 
         <v-card-text class="mt-4 mb-3 pb-1">
           <div v-if="typeOperation === 'subscription'">
-            <h4>Subscribe to premium plan</h4>
-            <p>
-              The amount charged will be based on the device usage.
-            </p>
+            <div data-test="subscription-description">
+              <h4>Subscribe to premium plan</h4>
+              <p>
+                The amount charged will be based on the current device usage.
+              </p>
+              <div>
+                <b>
+                  Estimated cost:
+                </b>
+                {{ currentQuantity }} devices : {{ priceEstimator(currentQuantity)
+                }} / month
```

Comment:
You could put this calculation on one line. Looks better.

{{ priceEstimator(currentQuantity) }} / month